libevmasm/KnownState.cpp

```diff
@@ -93,9 +93,15 @@ KnownState::StoreOperation KnownState::feedItem(AssemblyItem const& _item, bool
 		// can be ignored
 	}
 	else if (_item.type() == AssignImmutable)
+	{
 		// Since AssignImmutable breaks blocks, it should be fine to only consider its changes to the stack, which
-		// is the same as POP.
-		return feedItem(AssemblyItem(Instruction::POP), _copyItem);
+		// is the same as two POPs, i.e. the end of the else case below, only with _item.deposit() == -2.
```

Comment:
Why not feed two POPs?

Reply:
Doesn't fit in the current interface... it returns a ``StoreOperation``, not a list of them or anything... maybe something like
```
feedItem(AssemblyItem(Instruction::POP), _copyItem); // discard the return value
return feedItem(AssemblyItem(Instruction::POP), _copyItem);
```
would be fine... but I'm not sure, if it would be :-).

Looking at it: yeah, it actually should - the return value doesn't contain anything special, I'll change it.

Reply:
Changed.